platform/app/src/routes/Mode/Mode.tsx

```diff
@@ -329,6 +329,12 @@ export default function ModeRoute({
         commandsManager,
       });
 
+      const disableEditing = extensionManager._appConfig?.disableEditing;
+      if (disableEditing) {
+        const { toolbarService } = servicesManager.services;
```

Comment:
This absolutely doesn't belong here - really, the right way to do this is to create a new mode that simply doesn't have the editing tools.  That would obviate all the rest of this too, and would become obvious as to how to handle it.

Reply:
I don't want to put it here either. The requirement is to disable the editing tools based on config. Where is the best place to do that, considering the user just want to change the config, not create a new mode...

Reply:
I put it here because it's the best place I know that works for every mode created.

Reply:
@rodrigobasilio2022 I think for this issue requirement: ```Disable tools (tool by tool basis) e.g. never enable brush``` @wayfarer3130 is right, we can disable tools (tool by tool basis) by just not including them in a new mode.

Reply:
Please update the requirements in the issue page

Reply:
We should sync to discuss the requirements for IDC mode, since there are certain behaviours and tools that IDC don't want or wants and is not enabled/disabled in OHIF right now. But the correct way moving forward is IDC having its own mode (or even set of modes)

Reply:
I removed the modifications in this file until further discussion